source/API_Reference/Web_API_v3/cancel_scheduled_send.apiblueprint

```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+weight: 0
+title: Cancel Scheduled Sends
+navigation:
+  show: true
+---
+
+FORMAT: 1A
+
+# Cancellation of Scheduled Sends
+Cancelling scheduled send
+
+# Group Batch IDs
+
+If you set the [SMTPAPI header batch_id]({{root_url}}/API_Reference/SMTP_API/scheduling_parameters.html),
+it allows you to then cancel a scheduled send based on that batch_id by calling the [Cancel Scheduled Send endpoint]({{root_url}}/API_Reference/Web_API_v3/cancel_scheduled_send.html#Cancel-Scheduled-Sends)
+
+## Batch ID Generation [/mail/batch]
+Generate a new Batch ID to associate with scheduled sends
+
+### Generate Batch ID [POST]
+
++ Request
+
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "batch_id": "YOUR_BATCH_ID"
+            }
+
+## Validate Batch ID [/mail/batch/]
+
+### Validate Batch ID [GET]
+
+Validate whether or not a batch id is valid
+
++ Request
+
++ Response 200 (application/json)
+
+    + Body
+
+             {
+                "batch_id": "YOUR_BATCH_ID"
+             }
+
++ Response 400 (application/json)
+
+    + Body
+
+             {
+                "errors":[{"message": "invalid batch id"}]
+             }
+
+
+# Group Cancel Scheduled Sends
+
+The Cancel Scheduled Sends feature allows the customer to cancel a
+scheduled send based on a [Batch ID]({{root_url}}/API_Reference/SMTP_API/scheduling_parameters.html)
+included in the SMTPAPI header.
+
+## Manage user scheduled sends [/user/scheduled_send]
+
+A CRUD for managing user scheduled sends
+
+### Retrieve scheduled send [GET]
+
+Get cancel/paused scheduled send information.
+
++ Request
+   + Body
+
+            {
+               "batch_id": "YOUR_BATCH_ID"
+            }
+
++ Response 200 (application/json)
+
+   + Body
+
+            {
+               "batch_id": "YOUR_BATCH_ID",
+               "status": "cancel"
+            }
+
+
+
+### Update user scheduled send information [PATCH]
+
+Update the status of a scheduled send.
+
++ Request
+
+   + Body
+
+           {
+               "batch_id": "YOUR_BATCH_ID",
+               "status": "pause"
+           }
+
++ Response 204 (application/json)
+
++ Response 404
+
+### Cancel or pause a scheduled send [POST]
+
+Cancel or pause a scheduled send. If the maximum number of cancellations/pauses are added, HTTP 400 will
+be returned.
+
++ Request
+
+   + Body
+
+           {
+               "batch_id": "YOUR_BATCH_ID",
+               "status": "pause"
+           }
+
++ Response 201 (application/json)
+
++ Response 400
```

Comment:
This can go into the apiary blueprint - let me know when that's updated! 

+ Response 404 
  "" : "a status for this batch id exists, try PATCH to update the status"
  "" : "max limit reached"
  "batch_id" : "invalid batch id"